demo/index.html

```
@@ -157,73 +591,341 @@
+            document.querySelector("#raw-json-b").innerText = json_output_b
+            
+            # Compare versions if both are valid
+            if info_a.get('valid') and info_b.get('valid'):
+                try:
+                    version_a = AwesomeVersion(version_a_str)
+                    version_b = AwesomeVersion(version_b_str)
+                    
+                    comparison_html = '<div class="comparison-result '
+                    if version_a > version_b:
+                        comparison_html += 'greater"><i class="fas fa-chevron-up"></i>'
+                        comparison_html += f'{version_a_str} > {version_b_str}'
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)